### PR TITLE
fix(bug): clear tribute from Wanderer planets correctly

### LINKED ISSUE #10737: Wrong defense fleet assigned to Hai-captured Wanderer planets
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

The events “wanderers: unfettered invasion starts” and “wanderers: more systems lost” assign various Wanderer planets to the unfettered Hai. However, their defense fleet doesn’t change and still belongs to the Wanderers.

https://github.com/endless-sky/endless-sky/blob/c25fda412d72fc6c0b6553d95b3ca4c84fe5314f/data/wanderer/wanderers.txt#L776-L782

https://github.com/endless-sky/endless-sky/blob/c25fda412d72fc6c0b6553d95b3ca4c84fe5314f/data/wanderer/wanderers.txt#L863-L880

### Steps to Reproduce

1. Complete the Wanderer campaign to the point where the Eye opens.
1. Go to the Chy'chra system.
2. Demand tribute.
3. Note how you are attacked by a Wanderer fleet and the Wanderer systems get hostile as you destroy it.

### Expected Behavior

You should be attacked by a Hai fleet

### Screenshots

_No response_

### Link to save file

_No response_

### Operating System

Linux

### Game Source

Other (please describe)

### Game Version

0.10.9

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Bug Fix**

Fixes #10737

## Summary
Oops.
When I made #10221, I thought that all the events when the Wanderers evacuated were in `wanderers.txt`. It turns out that was incorrect, and I missed one event.
And I missed one in `wanderers.txt`.

### CODE REVIEW

File: data/wanderer/wanderers start.txt [hunk 1]
```diff
@@ -2457,6 +2457,22 @@ event "battle for sich'ka'ara ends"
 	planet "Kort Vek'kri"
 		attributes "unfettered"
 		spaceport `The Unfettered clearly do not know quite what to make of the Wanderer research station here that they have recently captured. Animals that were once part of various experiments are now wandering loose, and the Unfettered are making only sporadic efforts to keep them well-fed. Every once in a while one of them, driven by sheer hunger, tries to smash its way into one of the crates of military rations that the Unfettered have stacked up in the shade near the main landing pad.`
+		tribute clear
```
Comment (warp-core) [inline data/wanderer/wanderers start.txt:2460]: Use "remove tribute", not "tribute clear".

**Comment by warp-core** [comment]
Varu Mer'ek also needs its tribute removed at the time of "capture of ik'kara'ka".

File: data/wanderer/wanderers start.txt [hunk 2]
```diff
@@ -2457,6 +2457,22 @@ event "battle for sich'ka'ara ends"
...
 		spaceport `The Unfettered clearly do not know quite what to make of the Wanderer research station here that they have recently captured. Animals that were once part of various experiments are now wandering loose, and the Unfettered are making only sporadic efforts to keep them well-fed. Every once in a while one of them, driven by sheer hunger, tries to smash its way into one of the crates of military rations that the Unfettered have stacked up in the shade near the main landing pad.`
+		remove tribute
+
+# Patch mission for players who got these missions before 0.10.11 and didn't get tribute from Kort Vek'kri cleared.
+mission "Wanderers: Kort Vek'kri Tribute Patch"
+	landing
+	invisible
+	to offer
+		has "event: battle for sich'ka'ara ends"
+		not "event: kort vek'kri tribute clear"
```
Comment (Amazinite) [inline data/wanderer/wanderers start.txt:2468]: This condition seems like it'll never be used. You're not triggering this event anywhere else, and this mission can't repeat, so this condition will always be true prior to the first time that this mission offers.

Comment (TheGiraffe3) [reply data/wanderer/wanderers start.txt:2468]:
```suggestion
```

**Pull request merged**